test/units/utils/test_isidentifier.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division)
+__metaclass__ = type
+
+
+from ansible.utils.vars import isidentifier
+
+
+# Originally posted at: http://stackoverflow.com/a/29586366
+
+def test_keywords_not_in_PY2():
+    assert not isidentifier('True')
```

Comment:
```suggestion
    assert not isidentifier(u'True')
```
if it handles byte strings too, would be cool to see tests for it (and non-ascii sybmols)